girder/api/describe.py

```diff
@@ -436,7 +436,7 @@ def __init__(self, templatePath=None):
         self.vars = {
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder - REST API Documentation',
+            'brandName': ModelImporter.model('setting').get(constants.SettingKey.BRAND_NAME),
```

Comment:
Ok I just see that !
I will try to figure out which is the best way to prevent this issue